smpp/logger/logger.go

```diff
@@ -0,0 +1,25 @@
+package logger
+
+import (
+	"github.com/Sirupsen/logrus"
```

Comment:
Don't think it's worth the CPU of converting logs to JSON. Also don't like the idea of having an unnecessary dependency that will conflict with other loggers. Since this is a library we shouldn't impose any specific logging library other than stdlib.